atomic-core/copy.php

```diff
@@ -84,7 +84,7 @@ function resizeIframe(obj) {
         </li>
 
 
-    <?php } ?>
+    <?php }) ?>
```

Comment:
Unrelated fix, my editor was complaining about this. I don't think this page is actually used in practice.